.github/workflows/integrationTest.yml

```
@@ -971,4 +982,80 @@ jobs:
 #          max_attempts: 3
 #          timeout_minutes: 8
 #          retry_wait_seconds: 5
-#          command: cd terraform/ec2/linux && terraform destroy --auto-approve
\ No newline at end of file
+#          command: cd terraform/ec2/linux && terraform destroy --auto-approve
+
+  EC2AcceptanceIntegrationTest:
+    needs: [ GenerateTestMatrix, StartLocalStack, MakeBinary ]
+    name: 'EC2AcceptanceIntegrationTest'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arrays: ${{ fromJson(needs.GenerateTestMatrix.outputs.ec2_acceptance_matrix) }}
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: ${{env.CWA_GITHUB_TEST_REPO_NAME}}
+          ref: ${{env.CWA_GITHUB_TEST_REPO_BRANCH}}
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ env.TERRAFORM_AWS_ASSUME_ROLE }}
+          aws-region: us-west-2
+
+      - name: Cache if success
+        id: ec2-linux-integration-test
```

Looks like this should be `ec2-acceptance-test` since that's what you're using on line 1025 for the cache hit check.

```suggestion
        id: ec2-acceptance-test
```